lnwire/shutdown.go

```diff
@@ -82,6 +87,13 @@ func (s *Shutdown) Decode(r io.Reader, pver uint32) error {
 		s.ShutdownNonce = tlv.SomeRecordT(musigNonce)
 	}
 
+	// Set the custom records field to the remaining TLV records.
+	customRecords, err := NewCustomRecordsFromTlvTypeMap(typeMap)
```

Comment:
We need to delete the shutdown nonce from the `typeMap` above, otherwise it lands in the custom records as well.